sotodlib/site_pipeline/update_det_cal.py

```diff
@@ -621,26 +627,10 @@ def find_correction_results(band, chan, dset):
                 cal.phase_to_pW = pA_per_phi0 / (2 * np.pi) / cal.s_i * cal.polarity
 
             # Add bandpass informaton from bias group
-            bgs_lowband = [0, 1, 4, 5, 8, 9]
-            bgs_highband = [2, 3, 6, 7, 10, 11]
-
-            tube_flavor = am.obs_info.tube_flavor
-            if tube_flavor == 'mf':
-                lowband_str = 'f090'
-                highband_str = 'f150'
-
-            elif tube_flavor == 'uhf':
-                lowband_str = 'f220'
-                highband_str = 'f280'
-
-            elif tube_flavor == 'lf':
-                lowband_str = 'f030'
-                highband_str = 'f040'
-
-            if cal.bg in bgs_lowband:
-                cal.bandpass = lowband_str
-            elif cal.bg in bgs_highband:
-                cal.bandpass = highband_str
+            if cal.bg in bgs['lb']:
+                cal.bandpass = band_str[tube_flavor]['lb']
+            elif cal.bg in bgs['hb']:
+                cal.bandpass = band_str[tube_flavor]['hb']
```

Comment:
If referencing globals I think this needs to be capitalized variable names here.

Reply:
Yeah sorry, missed this earlier